### PR TITLE
manually merged Carlos' tutorial views, and fixed groupObj crashing b…

### PR DESCRIPTION
**Pull request by wrenliang**
…ug. Also updated endpoints to AWS

### CODE REVIEW

File: iOS/dough-bros-ios/View/Add Group/AddGroupViewController.swift [hunk 1]
```diff
@@ -75,18 +75,19 @@ class AddGroupViewController: UIViewController, UITextFieldDelegate {
...
         }
         
-        let detailedGroupsVC = GroupDetailsViewController()
-        /// TODO: fetch GroupObj by newGroupID, and set detailedGroupsVC.groupObc
+//        let detailedGroupsVC = GroupDetailsViewController()
+//        /// TODO: fetch GroupObj by newGroupID, and set detailedGroupsVC.groupObc
         
         navigationController?.popViewController(animated: true)
-        navigationController?.pushViewController(detailedGroupsVC, animated: true)
+//        navigationController?.pushViewController(detailedGroupsVC, animated: true)
```
Comment (wrenliang) [inline iOS/dough-bros-ios/View/Add Group/AddGroupViewController.swift:90]: temporary fix to force unwrap of groupObj inside detailedGroupsVC causing app to crash. will need to update with return value of createGroup API call

**Review by Harin329 — COMMENTED** [review]
Is the tutorial being triggered? or is that a todo. Also, the endpoint seems to be working but no results are actually returning unless tables are all empty right now. If these are all the case, then looks good to me!

**Comment by wrenliang** [comment]
> Is the tutorial being triggered? or is that a todo. Also, the endpoint seems to be working but no results are actually returning unless tables are all empty right now. If these are all the case, then looks good to me!

No I left the tutorial out for now. We can fit it into the flow later, once there is real info on the tutorial pages. Which login did you try with? I made logins for all the frontend devs (firstname@doughbros.com and Doughbros123!), so yours should be harin@doughbros.com

**Comment by wrenliang** [comment]
Also you can MySQL workbench into our remote DB.
Hostname: ec2-34-214-245-195.us-west-2.compute.amazonaws.com
Port: 3307

**Review by Harin329 — APPROVED** [review]

**Pull request merged**